controllers/feeds/deletePost.js

```diff
@@ -25,7 +25,12 @@ module.exports = async (req, res) => {
   }
 
   try {
-    const getstreamDeleteResult = await getstreamService.deleteFeedById(token, 'user_excl', postId);
+    const getstreamDeleteResult = await getstreamService.deleteFeedById({
+      feedGroup: 'user_excl',
+      activityId: postId,
+      userId,
+      token
+    });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good as they improve the readability and maintainability of the code by passing an object to the `deleteFeedById` function instead of individual arguments. However, it's important to ensure that all places where this function is called have been updated to match this new signature. Also, make sure that the `userId` and `token` are always available in the context where this function is called.
```diff
- const getstreamDeleteResult = await getstreamService.deleteFeedById(token, 'user_excl', postId);
+ const getstreamDeleteResult = await getstreamService.deleteFeedById({
+   feedGroup: 'user_excl',
+   activityId: postId,
+   userId,
+   token
+ });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->